tests/unit/tree/test_repo.py

```diff
@@ -46,6 +46,11 @@ def test_open_dirty_no_hash(tmp_dir, dvc):
         assert fobj.read() == "file"
 
 
+def test_commit_granularity(tmp_dir, scm, dvc):
+    tmp_dir.dvc_gen("foo", "foo", commit="add files")
+    dvc.commit("foo")
```

Comment:
Seems like this should be in `tests/func/test_commit.py`

Also, need to better check that commit actually works.

Reply:
Not sure if my current changes are sufficient